src/app/courses/add-courses/courses-add.component.html

```diff
@@ -1,102 +1,109 @@
-    </mat-form-field>
-    <mat-form-field>
-      <input matInput i18n-placeholder placeholder="Location" formControlName="location">
-    </mat-form-field>
-  </div>
+      <div *ngIf="!showDaysCheckBox">
+        <label i18n>Day</label>
+        <mat-checkbox (change)="onDayChange(day, $event.checked)" *ngFor="let day of days" class="margin-lr">{{day}}</mat-checkbox>
+      </div>
 
-  <div>
-    <label i18n>Background Color</label>
-    <input type="color" i18n-placeholder placeholder="Background Color" formControlName="backgroundColor" mat-icon>
-    <mat-error><planet-form-error-messages [control]="courseForm.controls.backgroundColor"></planet-form-error-messages></mat-error>
-    <label i18n>Foreground Color</label>
-    <input type="color" i18n-placeholder placeholder="Foreground Color" formControlName="foregroundColor" mat-icon >
-    <mat-error><planet-form-error-messages [control]="courseForm.controls.foregroundColor"></planet-form-error-messages></mat-error>
-  </div>
+      <mat-grid-list cols="3">
+        <mat-form-field>
+          <input matInput type="time" i18n-placeholder placeholder="Start Time" formControlName="startTime">
+          <mat-error><planet-form-error-messages [control]="courseForm.controls.startTime"></planet-form-error-messages></mat-error>
+        </mat-form-field>
+        <mat-form-field>
+          <input matInput type="time" i18n-placeholder placeholder="End Time" formControlName="endTime">
+          <mat-error><planet-form-error-messages [control]="courseForm.controls.endTime"></planet-form-error-messages></mat-error>
+        </mat-form-field>
+        <mat-form-field>
+          <input matInput i18n-placeholder placeholder="Location" formControlName="location">
+        </mat-form-field>
+      </mat-grid-list>
 
-  <div>
-    <button type="submit" mat-raised-button color="primary" i18n>Submit</button>
-    <button type="button" mat-raised-button color="warn" (click)="cancel()" i18n>Cancel</button>
-  </div>
-</form>
+      <mat-grid-list cols="3">
+        <label i18n>Background Color</label>
+          <input type="color" i18n-placeholder placeholder="Background Color" formControlName="backgroundColor" mat-icon>
+        <mat-error><planet-form-error-messages [control]="courseForm.controls.backgroundColor"></planet-form-error-messages></mat-error>
+        <label i18n>Foreground Color</label>
+          <input type="color" i18n-placeholder placeholder="Foreground Color" formControlName="foregroundColor" mat-icon >
+        <mat-error><planet-form-error-messages [control]="courseForm.controls.foregroundColor"></planet-form-error-messages></mat-error>
+      </mat-grid-list>
 
+      <div>
+        <button type="submit" mat-raised-button color="primary" i18n>Submit</button>
+        <button type="button" mat-raised-button color="warn" (click)="cancel()" i18n>Cancel</button>
+      </div>
+    </form>
+  </div>
+</div>
 <!--FOR DEBUGGING (remove once view is final)-->
-<div>
+<!-- <div>
```

Comment:
km-course-valid is used by testcase so it needs to be there. If its removed test also needs to be updated.